src/ansys_sphinx_theme/__init__.py

```diff
@@ -395,12 +395,12 @@ def convert_pdf_to_png(pdf_path: pathlib.Path, output_dir: pathlib.Path, output_
         Name of the output PNG file.
     """
     try:
-        doc = pymupdf.open(pdf_path)  # open document
-        for page in doc:
-            pix = page.get_pixmap()
-            pix.save(output_dir / output_png)
-    except RuntimeError as e:
-        raise RuntimeError(f"Failed to convert PDF to PNG: {e}")
+        images = convert_from_path(pdf_path, 500)
```

Comment:
Can we do the import in the case we are interested on calling this function only? I am expecting that, since system dependencies are required, this might lead to failure on resolving imports even and not only when calling it. Maybe I'm wrong but I would constraint the import call to this function only

Reply:
yes,i will add that